config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,20 @@
+CarrierWave.configure do |config|
+  # 開発環境用でひとまずminioベースの設定
+  config.fog_provider = 'fog/aws'
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['MINIO_ROOT_USER'],
+    aws_secret_access_key: ENV['MINIO_ROOT_PASSWORD'],
+    region: ENV['MINIO_REGION'],
+    endpoint: ENV['MINIO_ENDPOINT'],
+    path_style: true
+  }
+  config.fog_public = false
+  config.fog_authenticated_url_expiration = 180
+  config.fog_attributes = {'Cache-Control' => 'public, max-age=86400'}
+  config.remove_previously_stored_files_after_update = false
+  config.fog_directory = ENV['MINIO_BUCKET']
+  config.asset_host = "#{ENV['MINIO_ENDPOINT']}#{ENV['MINIO_BUCKET']}"
```

Comment:
ここもこれぐらいのほうが、良いかと思ってます。
とは言え考え方次第なので、疑問点あれば聞いていただけると幸いです。
```suggestion
  config.fog_directory = ENV['FOG_DIRECTORY']
  config.asset_host = "#{ENV['FOG_ASSET_HOST']}"
```

Reply:
修正自体はしたのですが、今後のために質問させてください！
環境変数の命名自体を設定項目と合わせて一対一にするのが読むときにはわかりやすいと思うのですが、例えばバケット名が変わった場合、変更前のMINIO_BUCKETを使い回すやり方に対して修正すべき環境変数が増えることになってしまうかと思います。
読みやすさと変更しやすさの優先順位は、変更した際の負担などをみつつバランスで判断ということになるでしょうか？
もし何か基本的な方針があれば教えていただけると助かります🙇

Reply:
@nos0611 
バランスで判断ですかね。
minioの設定は一度決めたら変えないと思います。

あと環境変数はそもそも「読みやすさ」「変更しやすさ」のために使う変数ではないということも一応意識しておいてください。あくまでも環境の変化に耐えるための変数ですね。

Reply:
そもそも通常の変数とは考え方自体が違うんですね！
ありがとうございます、よくわかりました！